### ISSUE TITLE
[CI] Third-party tests for repository-s3 fails

### ISSUE THREAD

**Issue by cbuescher** [elastic/elasticsearch#32192]
Saw two instances of this already today:

https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+master+third-party-tests/25/console
https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+6.x+third-party-tests/24/console

No reproduce like, failure seems to already be the gradle invocation:

```
10:43:22 > Configure project :plugins:repository-s3
10:43:22 Evaluating project ':plugins:repository-s3' using build file '/var/lib/jenkins/workspace/elastic+elasticsearch+master+third-party-tests/plugins/repository-s3/build.gradle'.
10:43:22 
10:43:22 FAILURE: Build failed with an exception.
10:43:22 
10:43:22 * Where:
10:43:22 Build file '/var/lib/jenkins/workspace/elastic+elasticsearch+master+third-party-tests/plugins/repository-s3/build.gradle' line: 112
10:43:22 
10:43:22 * What went wrong:
10:43:22 A problem occurred evaluating project ':plugins:repository-s3'.
10:43:22 > not all options specified to run against external S3 service
```

Earlier in the build logs:
```
10:43:03 A problem occurred evaluating project ':plugins:repository-s3'.
10:43:03 > not all options specified to run against external S3 service
10:43:03 
10:43:03 * Try:
10:43:03 Run with --info or --debug option to get more log output. Run with --scan to get full insights.
10:43:03 
10:43:03 * Exception is:
10:43:03 org.gradle.api.GradleScriptException: A problem occurred evaluating project ':plugins:repository-s3'.
10:43:03 	at org.gradle.groovy.scripts.internal.DefaultScriptRunnerFactory$ScriptRunnerImpl.run(DefaultScriptRunnerFactory.java:92)
10:43:03 	at org.gradle.configuration.DefaultScriptPluginFactory$ScriptPluginImpl$2.run(DefaultScriptPluginFactory.java:204)
10:43:03 	at org.gradle.configuration.ProjectScriptTarget.addConfiguration(ProjectScriptTarget.java:77)
10:43:03 	at org.gradle.configuration.DefaultScriptPluginFactory$ScriptPluginImpl.apply(DefaultScriptPluginFactory.java:209)
10:43:03 	at org.gradle.configuration.BuildOperationScriptPlugin$1.run(BuildOperationScriptPlugin.java:61)
10:43:03 	at org.gradle.internal.operations.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:300)
10:43:03 	at org.gradle.internal.operations.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:292)
10:43:03 	at org.gradle.internal.operations.DefaultBuildOperationExecutor.execute(DefaultBuildOperationExecutor.java:174)
10:43:03 	at org.gradle.internal.operations.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:90)
10:43:03 	at org.gradle.internal.operations.DelegatingBuildOperationExecutor.run(DelegatingBuildOperationExecutor.java:31)
10:43:03 	at org.gradle.configuration.BuildOperationScriptPlugin.apply(BuildOperationScriptPlugin.java:58)
10:43:03 	at org.gradle.configuration.project.BuildScriptProcessor.execute(BuildScriptProcessor.java:41)
10:43:03 	at org.gradle.configuration.project.BuildScriptProcessor.execute(BuildScriptProcessor.java:26)
10:43:03 	at org.gradle.configuration.project.ConfigureActionsProjectEvaluator.evaluate(ConfigureActionsProjectEvaluator.java:34)
10:43:03 	at org.gradle.configuration.project.LifecycleProjectEvaluator$EvaluateProject.run(LifecycleProjectEvaluator.java:105)
10:43:03 	at org.gradle.internal.operations.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:300)
10:43:03 	at org.gradle.internal.operations.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:292)
10:43:03 	at org.gradle.internal.operations.DefaultBuildOperationExecutor.execute(DefaultBuildOperationExecutor.java:174)
10:43:03 	at org.gradle.internal.operations.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:90)
10:43:03 	at org.gradle.internal.operations.DelegatingBuildOperationExecutor.run(DelegatingBuildOperationExecutor.java:31)
10:43:03 	at org.gradle.configuration.project.LifecycleProjectEvaluator.evaluate(LifecycleProjectEvaluator.java:68)
10:43:03 	at org.gradle.api.internal.project.DefaultProject.evaluate(DefaultProject.java:682)
10:43:03 	at org.gradle.api.internal.project.DefaultProject.evaluate(DefaultProject.java:138)
10:43:03 	at org.gradle.execution.TaskPathProjectEvaluator.configure(TaskPathProjectEvaluator.java:35)
10:43:03 	at org.gradle.execution.TaskPathProjectEvaluator.configureHierarchy(TaskPathProjectEvaluator.java:62)
10:43:03 	at org.gradle.configuration.DefaultBuildConfigurer.configure(DefaultBuildConfigurer.java:41)
10:43:03 	at org.gradle.initialization.DefaultGradleLauncher$ConfigureBuild.run(DefaultGradleLauncher.java:266)
10:43:03 	at org.gradle.internal.operations.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:300)
10:43:03 	at org.gradle.internal.operations.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:292)
10:43:03 	at org.gradle.internal.operations.DefaultBuildOperationExecutor.execute(DefaultBuildOperationExecutor.java:174)
10:43:03 	at org.gradle.internal.operations.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:90)
10:43:03 	at org.gradle.internal.operations.DelegatingBuildOperationExecutor.run(DelegatingBuildOperationExecutor.java:31)
10:43:03 	at org.gradle.initialization.DefaultGradleLauncher.configureBuild(DefaultGradleLauncher.java:179)
10:43:03 	at org.gradle.initialization.DefaultGradleLauncher.doBuildStages(DefaultGradleLauncher.java:138)
10:43:03 	at org.gradle.initialization.DefaultGradleLauncher.executeTasks(DefaultGradleLauncher.java:121)
10:43:03 	at org.gradle.internal.invocation.GradleBuildController$1.call(GradleBuildController.java:77)
10:43:03 	at org.gradle.internal.invocation.GradleBuildController$1.call(GradleBuildController.java:74)
10:43:03 	at org.gradle.internal.work.DefaultWorkerLeaseService.withLocks(DefaultWorkerLeaseService.java:152)
10:43:03 	at org.gradle.internal.work.StopShieldingWorkerLeaseService.withLocks(StopShieldingWorkerLeaseService.java:38)
10:43:03 	at org.gradle.internal.invocation.GradleBuildController.doBuild(GradleBuildController.java:96)
```

**Comment by cbuescher** [comment]
@vladimirdolzhenko I just saw #31918 was merged recently, could this be somehow related?

**Comment by ywelsch** [comment]
Fixed by #32315
